dev/com.ibm.ws.rest.handler.validator_fat/publish/servers/validator-customLoginModule-Server/bootstrap.properties

```diff
@@ -13,4 +13,5 @@ com.ibm.ws.logging.trace.specification=*=info:rest.configbased=all:rest.validati
 com.ibm.ws.logging.max.file.size=0
 ds.loglevel=debug
 derby.connection.requireAuthentication=true
-derby.user.dbuser=dbpass
\ No newline at end of file
+derby.user.dbuser=dbpass
+websphere.java.security.exempt=true
```

Comment:
@anjumfatima90 Let's see if we can remove bootstrap property websphere.java.security.exempt=true and alleviate the j2 security policy violations by declaring the required permissions.

Reply:
Removed the bootstrap property and added java permission in server.xml